### ISSUE TITLE
open notexists.txt = panic

### ISSUE THREAD

**Issue by indeyets** [nushell/nushell#516]
tried to open file which does not exist. got this:

```
thread 'main' panicked at '
called `Result::unwrap()` on an `Err` value: ShellError
{ error: Diagnostic(ShellDiagnostic
  { diagnostic: Diagnostic
    { severity: Error, code: None, message: "File could not be opened", labels: 
      [Label { span: Span { start: 5, end: 12 }, message: Some("file not found"), style: Primary }]
    }
  }), cause: None
}', src/libcore/result.rs:1084:5
```

**Comment by wycats** [comment]
#438

**Comment by wycats** [comment]
I'm looking into this together with all of the other unexpected panics.

Merged this issue into #438. Closing for hygiene.
